twake/frontend/src/app/components/ChatUploads/ChatUploadService.ts

```diff
@@ -22,6 +21,10 @@ export class ChatUploadService {
     this.handler = handler;
   }
 
+  private setCounter({ total, completed }: { total: number; completed: number }) {
+    this.counter = { total, completed };
```

Comment:
Why don't you rely on recoil state for this? With the hook from my previous comment it will be easy to achieve

Reply:
Already discussed with Stephane, counter will not exist anymore and rely on existing state

Reply:
Removed 👍🏻